pom.xml

```diff
@@ -73,7 +73,7 @@
           <showWarnings>true</showWarnings>
           <compilerArgs>
             <arg>-Werror</arg>
-            <arg>-Xlint:all</arg>
+            <arg>-Xlint:all,-processing</arg>
```

Comment:
This appears to be due to some dependency, but I'm not sure which one. Basically, there's a weirdness in the way Java 8 assumes that an annotation processor must understand *all* annotations rather than just the ones it cares about. The details and the workaround are described here:

https://github.com/spring-projects/spring-boot/issues/6421